modules/installation-creating-gcp-shared-vpc-ingress-firewall-rules.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-user-infra-vpc.adoc
+
+[id="installation-creating-gcp-shared-vpc-ingress-firewall-rules_{context}"]
+= Creating ingress firewall rules for a shared VPC in GCP
```

Comment:
This section seems duplicate with Creating cluster-wide firewall rules for a shared VPC in GCP. Please check.

Reply:
@shellyyang1989, it looks like the options in the commands are different, and I am not sure if they're trying to do exactly the same thing or not.

I don't work with the devs who work on ingress, so you know more about this than I do. What can I do today to make this work for GA, and will you file a bug for what you think we need to figure out to make this section right?

Reply:
@kalexand-rh I don't find the command lines are different. If you're seeing any difference, please let me know.